Changelog.md

```diff
@@ -3,6 +3,7 @@
 - The remote property on the electron mock was noch mocked correctly
 ### Changed
 - Render all panes of the App, but only the current is visible
+- Updated confirmation dialog style
```

Comment:
4.8.5 is already tagged, so this changelog entry should go under a new heading (either “Unreleased” or give it a new number like 4.8.6 and also bump the number in the `package.json`)

Reply:
ok